src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java

```diff
@@ -376,6 +374,26 @@ public boolean canAdministerProject(String projectID, String organizationId) {
         return false;
     }
 
+    public boolean canAdministerProject(FeedSource feedSource) {
```

Comment:
I think it'd make more sense to have this method correspond to the model:

```suggestion
    public boolean canAdministerFeedSource(FeedSource feedSource) {
```

Reply:
The `canManageFeed` method actually evaluates that permission. This (and the other overloaded methods) just use the model object to grab its project ID and actually check whether the user can administer the project.

For this reason, I think all of these should stay how they are. I don't want folks confused about the permission they're actually getting back.